apps/job-service/src/jobs/delayed-jobs.controller.ts

```diff
@@ -39,8 +39,15 @@ export class DelayedJobsController {
       order: [["createdAt", "DESC"]]
     });
```

Comment:
It would be best to use includes here to avoid extra queries. In the `findAll`:
```
include: [
  { association: "entityProject" },
  { association: "entitySite" }
]
```

For bonus points you can limit the columns that are fetched by adding `attributes: ["name"]` to the include.

In your local env, please make sure it's using a `LEFT OUTER JOIN` for those includes. I _think_ it switches to an `INNER JOIN` when you add a `where` to the include (based on something I was seeing on my most recent v3 branch), so you should be good to go, but if it's using an inner join, you can add `required: false` to the include to get it to switch to a left outer join.

Reply:
added this includes